### PR TITLE
Feature/deputy mayor data

### PR DESCRIPTION
**Pull request by paulmwatson**

### CODE REVIEW

File: here_for_change/municipalities/views.py [hunk 1]
```diff
@@ -40,8 +45,141 @@ class WardDetail(DetailView):
...
+                                'feedback': {'positive': 5, 'negative': 0}}
+                        },
+                        'phone': {
+                            'value': '+27(0)284255513',
+                            'type': 'phone',
+                            'updated': '2022/09/12 16:57',
+                            'feedback': {'positive': 5, 'negative': 0}},
+                    }
+                },
+                'secretary_deputy_mayor': {
```
Comment (paulmwatson) [inline here_for_change/municipalities/views.py:160]: @Hazmid data for secretary of the deputy mayor

File: here_for_change/municipalities/templates/municipalities/ward_detail.html [hunk 2]
```diff
@@ -159,7 +159,7 @@ <h2 class="text-2xl md:text-3xl benfritz text-white text-center">
               <i class="fa-solid fa-envelope"></i>
             </span>
             <span class="text-sm mr-1 md:mr-5 hover:bg-gray-200 p-2 rounded-md cursor-pointer whitespace-nowrap">
-              {{content.councillor.contacts.email.primary.value}}
+              {{content.ward.councillor.contacts.email.primary.value}}
```
Comment (paulmwatson) [inline here_for_change/municipalities/templates/municipalities/ward_detail.html:188]: @Hazmid note that the data has moved up a level so it was `content.councillor.contacts.email.primary.value` but is now `content.ward.councillor.contacts.email.primary.value`. This allows for `content.municipality.office_of_the_deputy_mayor.secretary_deputy_mayor.name.value`

Comment (Hazmid) [reply here_for_change/municipalities/templates/municipalities/ward_detail.html:188]: okay, noted.

**Comment by Hazmid** [comment]
@paulmwatson  Dummy data has been formatted and rendered on template.

File: here_for_change/municipalities/templates/municipalities/toc_summary.html [hunk 3]
```diff
@@ -1607,3 +1610,17 @@
     </div>
   </div>
 </div>
+<script>
```
Comment (paulmwatson) [inline here_for_change/municipalities/templates/municipalities/toc_summary.html]: Same here, no JS in the page please.

**Pull request merged**